### PR TITLE
Added Multinomial, NegativeMultinomial Distribution

### PR DESCRIPTION
**Pull request by czgdp1807**
#### References to other Issues or PRs

Extends https://github.com/sympy/sympy/pull/16576/
Don't close any of the two PRs.

#### Brief description of what is fixed or changed
Multinomial Distribution has been added to
sympy.stats.joint_rv_types. There are import
issues remaining to be resolved.

#### Other comments
N/A

#### Release Notes

* stats
  * Addition of Multinomial Distribution to `sympy.stats.joint_rv_types`

### CODE REVIEW

**Comment by czgdp1807** [comment]
@Upabjojr @sidhantnagpal Please review this PR.

File: sympy/stats/joint_rv_types.py [hunk 1]
```diff
@@ -253,3 +248,138 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
...
+    >>> from sympy.stats.joint_rv import marginal_distribution
+    >>> from sympy.stats.joint_rv_types import NegativeMultinomial
+    >>> from sympy import symbols
+    >>> x1, x2, x3 = symbols('x1, x2, x3', nonnegative=True, integer=True)
+    >>> p1, p2, p3 = symbols('p1, p2, p3', positive=True)
+    >>> N = NegativeMultinomial('M', 3, p1, p2, p3)
+    >>> N_c = NegativeMultinomial('M', 3, 0.1, 0.1, 0.1)
+    >>> density(N)(x1, x2, x3)
+    p1**x1*p2**x2*p3**x3*(-p1 - p2 - p3 + 1)**3*gamma(x1 + x2 + x3 + 3)/
+    (2*factorial(x1)*factorial(x2)*factorial(x3))
```
Comment (czgdp1807) [inline sympy/stats/joint_rv_types.py]: @sidhantnagpal  Lines 372 and 373 when written on a single line are getting too long. [Doctests](https://travis-ci.org/sympy/sympy/jobs/531121583) are failing due to writing the result in two lines. Is there something like `\` for docstrings?

**Comment by czgdp1807** [comment]
@sylee957 This PR belongs to `stats` module. Please update the labels.

File: sympy/stats/joint_rv_types.py (hunk 1 above)
Comment (sidhantnagpal) [reply sympy/stats/joint_rv_types.py]: Would it be possible to make use of `#doctest: +NORMALIZE_WHITESPACE`?

Comment (czgdp1807) [reply sympy/stats/joint_rv_types.py]: I exploited the property of  continuation in `()`, like,
```
>>> some function
(a + 
b)
```
The above worked well. Though found [this](https://docs.python.org/2/library/doctest.html#how-are-docstring-examples-recognized). Thanks for the comment.

**Pull request merged**